KarhooUISDK/API/KarhooUISDKConfiguration.swift

```diff
@@ -10,11 +10,17 @@ import KarhooSDK
 
 public protocol KarhooUISDKConfiguration: KarhooSDKConfiguration {
     func logo() -> UIImage
+    
+    func bookingMetadata() -> [String: Any]
```

Comment:
This can be a `var`

Reply:
I would suggest keeping it as a function:
- To be consistent with the existing properties we can use the function 
- Tried the `var` option, it did not work as expected. Happy to pair and try it (however because of the above reason, I am more inclined to keep it as a function)